R/pkg/R/mllib.R

```diff
@@ -1852,9 +1867,9 @@ summary.treeEnsemble <- function(model) {
 
 #  Get the summary of a Random Forest Regression Model
 
-#' @return \code{summary} returns a summary object of the fitted model, a list of components
-#'         including formula, number of features, list of features, feature importances, number of
-#'         trees, and tree weights
+#' @return \code{summary} returns summary information of the fitted model, which is a list.
+#'         The list of components includes \code{ans} (formula, number of features, list of features,
+#'         feature importances, number of trees, and tree weights).
```

Comment:
this should be something like
```
\code{formula} (something), \code{numFeatures} (something), \code{features} ....
```

`ans` is not the actual item or list of items. See https://github.com/apache/spark/pull/16150/files#diff-7ede1519b4a56647801b51af33c2dd18R1859